framework/CodeInterfaces/CobraTF/CTFinterface.py

```diff
@@ -11,13 +11,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+Modified by Alp Tezbasaran @ INL (6/21/2018)
+"""
+
 from __future__ import division, print_function, unicode_literals, absolute_import
 import warnings
 warnings.simplefilter('default',DeprecationWarning)
 from ctfdata import ctfdata
 from CodeInterfaceBaseClass import CodeInterfaceBase
+from GenericCodeInterface import GenericParser
+
 import os
-class CobraTF(CodeInterfaceBase):
+class CTF(CodeInterfaceBase):
```

Comment:
this changes the way we invoke the code interface from the RAVEN input file... why?